src/Features/Core/Portable/SymbolSearch/Windows/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -554,9 +554,17 @@ private async Task<XElement> TryDownloadFileAsync(IRemoteControlClient client, C
 
                 using var reader = XmlReader.Create(stream, settings);
 
-                var result = XElement.Load(reader);
-                await LogInfoAsync("Converting data to XElement completed", cancellationToken).ConfigureAwait(false);
-                return result;
+                try
+                {
+                    var result = XElement.Load(reader);
+                    await LogInfoAsync("Converting data to XElement completed", cancellationToken).ConfigureAwait(false);
+                    return result;
+                }
+                catch (XmlException e)
+                {
+                    await LogExceptionAsync(e, "Converting data to XElement failed", cancellationToken).ConfigureAwait(false);
+                    return null;
```

Comment:
i actually don't understand what change this would make.  this will still report the failure.  so we'll still get as many hits in prism for issues.

Reply:
From my understanding `LogExceptionAsync` is more explicit here then letting an exception propogate and eventually become a fault. Do we still end up reporting faults from the `ILogger` implementation here? Based on usage and a quick local test it didn't seem so but I could be mistaken here.

Reply:
All of these codepaths shoudl bubble all the way back up to: UpdateDatabaseInBackgroundWorkerAsync which has this:

```c#
                catch (Exception e) when (_service._reportAndSwallowExceptionUnlessCanceled(e, cancellationToken))
                {
                    // Something bad happened (IO Exception, network exception etc.).
                    // ask our caller to try updating again a minute from now.
                    //
                    // Note: we skip OperationCanceledException because it's not 'bad'.
                    // It's the standard way to indicate that we've been asked to shut
                    // down.
                    var delay = _service._delayService.ExpectedFailureDelay;
                    await LogExceptionAsync(e, $"Error occurred updating. Retrying update in {delay}", cancellationToken).ConfigureAwait(false);
                    return delay;
                }
```

Reply:
Is there a reason to double log in that instance?

Reply:
_reportAndSwallowExceptionUnlessCanceled is reporting the exception to prism as a fault.  LogException is logging to the VSLog service (e.g. ActivityLog.xml). to make it easier to understand what's happening locally in case of a problem.

Reply:
interesting. Maybe we should revisit this pattern somehow to make it easier to do or more explicit? 

I find our logging very confusing :)